### ISSUE TITLE
Clean Flash for Windows XP 64Bit cannot install IE Flash.

### ISSUE THREAD

**Issue by jonm58** [Skulltrail192/One-Core-API-Binaries#187]
![图片](https://github.com/Skulltrail192/One-Core-API-Binaries/assets/73036321/a410a1a9-ccf4-40f0-b77d-971d2e9e855c)
CleanFlashCommon.InstallException: Failed to register ActiveX plugin: error code 5

Error Code:
 ```
   at CleanFlashInstaller.Installer.RegisterActiveX(String filename)
   at CleanFlashInstaller.Installer.InstallFromArchive(SevenZipArchive archive, IProgressForm form, InstallFlags flags)
   at CleanFlashInstaller.Installer.Install(IProgressForm form, InstallFlags flags)
   at CleanFlashInstaller.InstallForm.<>c__DisplayClass15_0.<BeginInstall>b__0()
 ```

**Comment by Skulltrail192** [comment]
What is the IE version installed?

**Comment by jonm58** [comment]
> What is the IE version installed?

IE 9

**Comment by Snaky1a** [comment]
> IE 9

Internet Explorer 9 on Windows XP x64, really?

**Comment by jonm58** [comment]
> > IE 9
> 
> Internet Explorer 9 on Windows XP x64, really?

sorry

**Comment by Skulltrail192** [comment]
Fixed at version 3.0.5
